Cargo.toml

```diff
@@ -11,6 +11,7 @@ members = [
   "examples/chat/module/rust",
   "examples/hello_world/module/rust",
   "examples/machine_learning/module/rust",
+  "examples/private_information_retrieval/module/rust",
```

Comment:
BTW, should we rename this example to `trusted_information_retrieval` or something like that? perhaps start a thread on slack to discuss